src/App.tsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { useState } from "react";
+import UserHeader from "./components/header";
+import CharacterCard from "./components/characters";
+import { ContentStyle } from "./styled/style";
+
+import Wednesday from "./components/img/wednesday.jpg";
+import Erid from "./components/img/erid.jpg";
+import Thing from "./components/img/thing.jpg";
+import Tyler from "./components/img/tyler.jpg";
+import Xavier from "./components/img/xavier.jpg";
+import Larissa from "./components/img/larissa.jpg";
+
+type Character = {
+  id?: number;
+  name: string;
+  image: string;
+};
+
+const characters: Character[] = [
```

Comment:
Can we move this in a separate file for mock data

Reply:
Resolved